assets/src/globals.js

```diff
@@ -17,7 +19,7 @@ const user = Object.freeze({
 })
 
 const palette = JSON.parse(JSON.stringify(defaultPalette))
-if (mylaGlobals.primary_ui_color !== null) {
+if (mylaGlobals.primary_ui_color !== null && mylaGlobals.primary_ui_color !== undefined) {
```

Comment:
I think we could use `mylaGlobals.primary_ui_color != null` here instead, but this is a bit more explicit.

Reply:
I think you can use it as it makes the check simpler, you can add a comment saying checking for both null and undefined

Reply:
Done! Let me know what you think.